### DISCUSSION TITLE
Welcome to DART-Pipeline Discussions!

### DISCUSSION THREAD

**Discussion by jsbrittain** [kraemer-lab/DART-Pipeline#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jsbrittain** [comment]
Hi - I'm John Brittain, a Research Software Engineer engineer dealing with integration and data pipelines.

**Comment by rowannicholls** [comment]
Rowan Nicholls, also a Research Software Engineer in the same team as John. Working on the scripts that get integrated into John's GRAPEVNE framework

**Comment by Mougk** [comment]
Hi Hi: I am Moritz, an infectious disease modeler interested in how we can anticipate the control disease outbreaks. Looking to make our DART tool most useful for policy makers and other research groups around the world.

**Comment by prathyushsambaturu** [comment]
Hi, I am Prathyush, a postdoc in Moritz Kraemer's group at Oxford. My research interests lie at the intersection of computer science (network science, optimisation) and epidemiology.

**Comment by tuyenhn** [comment]
Hi, I'm Tuyen, an Oxford DPhil student based in the Mathematical Modelling Group, in Oxford University Clinical Research Unit (OUCRU), Vietnam. I have a background in computer science and experience in software engineering and cloud technologies. I'm currently focusing on building the dengue incidence forecast model for DART, by combining mechanistic models and machine learning models, as part of my thesis.

**Comment by Winter-23** [comment]
Hello, I am Iago, a postdoc in the Engineering Science group. I come from the meteorology/oceanographic field, and  I am working on the processing and evaluation of the meteorological data that is used in the dengue incidence forecast model.

**Comment by snsparrow** [comment]
Hi, I am Sarah, I lead the DART project and have a background in climate and meteorology in particular extreme weather events.
